nulog-inference-service/start-nulog-inference.py

```diff
@@ -21,8 +25,26 @@
 THRESHOLD = float(os.getenv("MODEL_THRESHOLD", 0.7))
 ES_ENDPOINT = os.environ["ES_ENDPOINT"]
 IS_CONTROL_PLANE_SERVICE = bool(os.getenv("IS_CONTROL_PLANE_SERVICE", False))
+IS_GPU_SERVICE = bool(os.getenv("IS_GPU_SERVICE", False))
 
 nw = NatsWrapper()
+es = AsyncElasticsearch(
+    [ES_ENDPOINT],
+    port=9200,
+    http_compress=True,
+    http_auth=("admin", "admin"),
```

Comment:
should not be setting auth details like this

Reply:
should be `http_auth=(ES_USERNAME, ES_PASSWORD)`

with

```
ES_USERNAME = os.getenv("ES_USERNAME", "admin")
ES_PASSWORD = os.getenv("ES_PASSWORD", "admin")
```

Reply:
good catch. fixed.